src/stp/new_skills/Chip.cpp

```diff
@@ -6,28 +6,28 @@
 
 namespace rtt::ai::stp {
 
-    void Chip::onInitialize() noexcept { }
+void Chip::onInitialize() noexcept {}
 
-    Status Chip::onUpdate(const rtt::ai::stp::SkillInfo &info) noexcept {
-        double chipVelocity = info.getKickChipVelocity();
+Status Chip::onUpdate(const StpInfo &info) noexcept {
+    double chipVelocity = info.getKickChipVelocity();
 
-        // Check if chip velocity is in range
-        if (chipVelocity < 0.0 || chipVelocity > Constants::MAX_KICK_POWER()) {
-            return Status::Failure;
-        }
+    // Check if chip velocity is in range
+    if (chipVelocity < 0.0 || chipVelocity > Constants::MAX_KICK_POWER()) {
+        return Status::Failure;
+    }
```

Comment:
Well, a skill can't really fail, can it? Maybe clamp

Reply:
I'll refactor the changes we discussed yesterday in a different PR